### PR TITLE
add moyo header

### PR DESCRIPTION
**Pull request by OksanaGrygorchuk**
https://OksanaGrygorchuk.github.io/layout_moyo-header/
https://OksanaGrygorchuk.github.io/layout_moyo-header/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,8 +7,53 @@
...
+          >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#sumsung" class="nav__link">Samsung</a>
+          </li>
```
Comment (nazarmatsevych) [inline src/index.html:30]:
```suggestion
          <li class="nav__item">
            <a href="#apple" class="nav__link is-active">Apple</a>
          </li>

          <li class="nav__item">
            <a href="#sumsung" class="nav__link">Samsung</a>
          </li>
```

add blank lines between multiple sibling blocks

File: src/style.css [hunk 2]
```diff
@@ -1 +1,71 @@
...
+.nav__link.is-active {
+  color: #00acdc;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active:hover {
+  color: #00acdc;
```
Comment (nazarmatsevych) [inline src/style.css:70]: It's a good practice to move repetitive colors to variables

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
Almost done, pls check my comments

File: src/style.css [hunk 3]
```diff
@@ -1 +1,69 @@
...
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  font-family: "Roboto", sans-serif;
```
Comment (slavanosatskyi) [inline src/style.css:45]: you have already defined it in body

**Review by slavanosatskyi — APPROVED** [review]

**Pull request closed without merging**